addons/mass_mailing/models/res_partner.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class Partner(models.Model):
+    _inherit = 'res.partner'
+    _mailing_enabled = True
```

Comment:
Strange that it's defined within mass_mailing for partner and not on the base module definition like other models.
But I guess it's an exception.

Reply:
Well I will not make bridge module just for that. When we have one (mass_mailing directly for partner) it is defined in it.

Reply:
Makes me think we have a bridge module for crm and sales. Will move definition in those modules.